lib/fluent/log.rb

```diff
@@ -67,8 +67,29 @@ def self.event_tags
       LEVEL_TEXT.map{|t| "#{LOG_EVENT_TAG_PREFIX}.#{t}" }
     end
 
+    # Create a unique path for each process.
+    #
+    # >>> per_process_path(:worker, 1, "C:/tmp/test.log")
```

Comment:
I thought `per_process_path("C:/tmp/test.log", :worker, 1)` was easier to understand because the order of the arguments is the same.

Reply:
Wow, indeed. Thanks for the advice.
Why wasn't it a match...

Reply:
Looks like we missed it from the start. I'll fix this. Thanks!

Reply:
Fixed

https://github.com/fluent/fluentd/compare/879eeecc1f1f4865e554be2f3057f8ad71cfe7ba..a9f511345a9adec2f4f5b78de8e741de3ef7de56